### PR TITLE
Use edu benefits data from api call on confirmation page

### LINKED ISSUE #3223: Use data from api call on confirmation page
The edu benefits confirmation page has hard coded values for the fields shown. This needs to be updated to real data from the api or the form.

### PR DESCRIPTION
**Pull request by jbalboni**
Closes #3223

This uses the live data that comes back from the api and populates the benefits fields from form data.

screen shot 2016-10-05 at 5 09 28 pm

### CODE REVIEW

**Comment by jbalboni** [comment]
@akainic Can you take a look?

File: src/js/edu-benefits/utils/helpers.js [hunk 1]
```diff
@@ -99,3 +99,25 @@ export function hasServiceBefore1978(data) {
 export function showRelinquishedEffectiveDate(benefitsRelinquished) {
   return benefitsRelinquished !== '' && benefitsRelinquished !== 'unknown';
 }
+
+export function getListOfBenefits(veteran) {
```
Comment (jbalboni) [inline src/js/edu-benefits/utils/helpers.js]: Maybe there's a better way to do this?

Comment (akainic) [inline src/js/edu-benefits/utils/helpers.js]: Good question... I think this works for now and we can always add a TODO to revisit this later.

**Review by akainic — APPROVED** [review]
Looks great! Feel free to merge when ready

File: schema [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit c4b69c3a5076d31e2d40d5ab7f053aab6bdbe938
+Subproject commit cdc48375d79cce57ceae86c1095e7a7c00c27324
```
Comment (jbalboni) [inline schema]: Yeah, this was intentional.

**Pull request merged**